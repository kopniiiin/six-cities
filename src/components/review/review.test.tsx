import * as React from "react";
import {configure, shallow} from "enzyme";
import * as Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";

import Review from "./review";

configure({adapter: new Adapter()});

const testMocks = {
  date: `2020-04-04`,
  text: `Good`,
  rating: 4,
  user: {
    name: `User`,
    photo: `photo`
  }
};

describe(`snapshot test: Review component`, () => {
  it(`should render correctly`, () => expect(
      toJson(shallow(<Review {...testMocks}/>))
  ).toMatchSnapshot());
});
