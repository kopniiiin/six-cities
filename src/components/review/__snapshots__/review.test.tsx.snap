// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test: Review component should render correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="User photo"
        className="reviews__avatar user__avatar"
        height="54"
        src="photo"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      User
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <StarRating
      blockClassName="reviews"
      isValueShown={false}
      value={4}
    />
    <p
      className="reviews__text"
    >
      Good
    </p>
    <time
      className="reviews__time"
      dateTime="2020-04-04"
    >
      April 4, 2020
    </time>
  </div>
</li>
`;
